translations/localify/ui.json

```diff
@@ -2287,5 +2287,40 @@
     "選択数": "Selected",
     "期限が切れたプレゼントは<color=#ff911c>自動的に削除</color>されます\n一度に受け取れるプレゼントは<color=#ff911c>100件</color>までです": "Expired presents will be <color=#ff911c>auto-deleted</color>\nYou can only receive <color=#ff911c>100 gifts</color> at a time",
     "受け取れるプレゼントはありません": "No presents",
-    "ピースを変換できます。\n変換で獲得したクローバーは\nアイテムと交換できますよ。": "Pieces can be converted. \nThe clovers obtained by conversion \ncan be used to buy items."
-}
+    "ピースを変換できます。\n変換で獲得したクローバーは\nアイテムと交換できますよ。": "Pieces can be converted. \nThe clovers obtained by conversion \ncan be used to buy items.",
+    "このイベントを再生しますか？": "Do you want to replay this event?",
+    "分岐は<color=#ff9900>ランダムで再生</color>されます": "Branches will be <color=#FF9900>replayed randomly</color>",
+    "トーク": "Talk",
+    "アプリ終了確認": "Application exit confirmation",
+    "アプリを終了してもよろしいですか？": "Are you sure you want to exit the application?",
+    "育": "Training",
+    "成": "<empty>",
+    "イ": "<empty>",
+    "ベン": "Event",
+    "ト": "<empty>",
+    "ギ": "Gallery",
+    "ャ": "<empty>",
+    "ラ": "<empty>",
+    "リ": "<empty>",
+    "ー": "<empty>",
+    "あら、\n今日はどんなご用ですか？": "Oh, \nwhat can I do for you today?",
+    "あら、今日はどんなご用ですか？": "Oh, what can I do for you today?",
+    "名鑑レベルが上がると\n報酬が手に入ります。\nがんばりましょうね！": "You will get rewards \nas your index level goes up. \nKeep up the good work!",
+    "名鑑レベルが上がると報酬が手に入ります。がんばりましょうね！": "You will get rewards as your index level \ngoes up. Keep up the good work!",
+    "歴代評価点を更新し、\n名鑑レベルを上げて\nいきましょう！": "Let's update your past \nrating scores and raise \nyour index level!",
+    "歴代評価点を更新し、名鑑レベルを上げていきましょう！": "Let's update your past rating scores \nand raise your index level!",
+    "共 通": "Common",
+    "ハルウララ登場！": "Haru Urara is here!",
+    "新年の抱負": "New Year's ambitions",
+    "初詣": "New Year's visit to the shrine",
+    "夏合宿（2年目）スタート！": "Summer training camp (2nd year) starts!",
+    "夏合宿（2年目）にて": "At the summer training camp (2nd year)",
+    "夏合宿（2年目）終了": "Summer training camp (2nd year) ends",
+    "夏合宿（3年目）スタート！": "Summer training camp (3rd year) starts!",
+    "夏合宿（3年目）終了": "Summer training camp (3rd year) ends",
+    "バレンタイン": "Valentine's Day",
+    "ファン感謝祭": "Fan festival",
+    "クリスマス": "Christmas",
+    "福引チャンス！": "Lottery chance!",
+    "デビュー戦にむけて": "Toward the debut match"
```

Comment:
Event names can be translated in mdb so that's where they should go. (same as earlier comment)
But atm we purposefully don't do this for compatibility with tools like UmaUmaCruise and other event helpers.
Thanks for translating them though, I suggest you keep them separate somewhere for later use and remove them from the PR for now.